packages/plugin-loansresearch-ui/src/components/LoansResearchForm.tsx

```
@@ -83,6 +88,18 @@
     }
   }, [queryParams]);
 
+  useEffect(() => {
+    if (customer?._id) {
+      setCustomerId(customer._id);
+    }
+  }, [customer]);
+
+  useEffect(() => {
+    if (queryParams && queryParams?.itemId) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)